app/tests/spec/models/email-resend.js

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+
+define([
+  'chai',
+  'models/email-resend',
+  'lib/constants'
+],
+function (chai, EmailResend, Constants) {
+  var assert = chai.assert;
+
+  describe('models/email-resend', function () {
+    var emailResend;
+
+    afterEach(function () {
+      emailResend = null;
+    });
+
+    describe('requestResend', function () {
+      it('increments the number of tries', function () {
+        emailResend = new EmailResend();
+        assert.equal(emailResend.get('tries'), 0);
+        emailResend.requestResend();
+        assert.equal(emailResend.get('tries'), 1);
+      });
+
+      it('calls onTriesExceeded ONCE when tries are exceeded', function () {
+        var numCalls = 0;
+
+        function onTriesExceeded() {
```

Comment:
You can use [sinon.spy](http://sinonjs.org/docs/#spies) to simplify the `numCalls` logic, since a spy has a `callCount` field.